### PR TITLE
Adds playerlist header and footer broadcasting

### PR DESCRIPTION
**Pull request by dImrich**
Notchian server does not implement this so I made it as simple as possible.
Maybe it would be even better to have it exported only in ClientHandle and let plugins handle the rest.

### CODE REVIEW

**Comment by dImrich** [comment]
I tested it and it works on 1.8, 1.9, 1.9.4, 1.10.2, 1.11.2, 1.12.2

File: Server/Plugins/APIDump/APIDesc.lua [hunk 1]
```diff
@@ -1584,6 +1584,26 @@ end
...
+								Type = "cCompositeChat",
+								Name = "a_Footer",
+							},
+						},
+						Returns =
+						{
+						},
+						Desc = "Sends the header and footer of the player list to the client."
+					},
+				},
```
Comment (tigerw) [inline Server/Plugins/APIDump/APIDesc.lua:1606]: Looks like an extra set of outer braces, and an extra Returns. Could you also delete the a_ from a_Header/a_Footer, and the same for the other API additions.

**Comment by tigerw** [comment]
Thanks, looks great. We can finally have in-game server branding xD

**Comment by tigerw** [comment]
Thanks!

**Comment by dImrich** [comment]
It's my pleasure, thanks for a quick review and merge.

**Pull request merged**